Modix/Modules/MoveMessageModule.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+using Modix.Services.CommandHelp;
+
+namespace Modix.Modules
+{
+    [Name("Move Message"), RequireUserPermission(GuildPermission.ManageMessages), HiddenFromHelp]
+    public class MoveMessageModule : ModuleBase
+    {
+        [Command("move"), Summary("Moves a message from one channel to another."), Remarks("Usage: `!move 12345 #foo`")]
+        public async Task Run([Remainder] string command)
```

Comment:
Let discord.net do the parsing for you. The parameters should be IMessage and ISocketChannel (I think). discord.net will figure out what to do to parse out the guild and message for you.